File: js_basics/ruby_versions/password.rb
password = "password"

counter = 0

loop do
  puts "Please enter password:"
  input = gets.chomp
  if input == password
    puts "You have logged in."
    exit
  end
  counter += 1
  if counter == 3 
    puts "You have been denied access."
    exit
  end
end

File: js_basics/password.js
var password = "password";

function enterPw() {
  for (var i = 0; i < 3; i++) {
    var input = prompt("Please enter password:");
    if (input === password) {
      console.log("You have succesfully logged in.");
      return;
    }
  }
  console.log("You have been denied access.");
}

enterPw();

